x/curating/client/rest/query.go

```diff
@@ -28,9 +28,11 @@ func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	).Methods("GET")
 }
 
+var ok bool
```

Comment:
this will create a global state and probably some weird issues, I think in order to pass the link you can do.

```go
return func(w http.ResponseWriter, r *http.Request) {
  var ok bool
  cliCtx, ok = rest.ParseQueryHeightOrReturnBadRequest(w, cliCtx, r)
```